webapp/tests/karma/ts/modules/tasks/tasks.component.spec.ts

```diff
@@ -278,4 +303,161 @@ describe('TasksComponent', () => {
       expect(component.listTrackBy(0, false)).to.equal(undefined);
     });
   });
+
+  describe('lineage and breadcrumbs', () => {
+    const bettysContactDoc = {
+      _id: 'user',
+      parent: {
+        _id: 'parent',
+        name: 'CHW Bettys Area',
+      },
+    };
+    const taskDocs = [
+      { _id: '1', emission: { _id: 'e1', dueDate: '2020-10-20' }, forId: 'a', owner: 'a' },
+      { _id: '2', emission: { _id: 'e2', dueDate: '2020-10-20' }, forId: 'b', owner: 'b' },
+    ];
+    const taskLineages = [
+      {
```

Comment:
This is breaking in a weird place. When breaking object please put every property in separate lines, like this:
```
{
  _id: ...
  lineage: [ ... ]
}
```
The idea is to have different data possibilities that can break your code. When writhing tests, think about what can break your code. Because you know users not always follow the happy path.

You can test this with 3 objects.
1. obj has lineage
2. obj lineage property is missing
3. obj lineage property is an empty array